extensions/SampleExtension/Helpers/FileHelper.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Text;
+using System.Text.Json;
+using Serilog;
+
+namespace SampleExtension.Helpers;
+
+public static class FileHelper
+{
+#pragma warning disable CS8603 // Possible null reference return.
```

Comment:
Is the PRAGMA just for the example?

Reply:
If it is, a comment should be added to indicate such.

Reply:
I'll be removing all of that, it isn't used.